views/partials/header.ejs

```diff
@@ -37,10 +37,10 @@
           <a class="nav-link btn btn-outline-primary my-1 my-xl-0" href="/user/login">Log in</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link btn btn-primary" href="/signup/candidate">Sign up as developer</a>
+          <a class="nav-link btn btn-primary" href="/signup/candidate">Sign up as a job seeker</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link btn btn-primary" href="/signup/recruiter">Sign up as employer</a>
+          <a class="nav-link btn btn-primary" href="/signup/recruiter">Sign up as recruiter</a>
```

Comment:
if we write **as a job seeker**, dont forget **as a recruiter.**
or maybe not to make a mistake "Job seeker signup / Recruiter sign up" as you wish
![image](https://user-images.githubusercontent.com/38938392/106354581-30309f80-62f3-11eb-8217-e94c95aa8dc1.png)